package.json

```
@@ -6,6 +6,7 @@
   "scripts": {
     "start": "node dist",
     "build": "ncc build --license LICENSE",
+    "cli": "ncc build src/cli.ts",
```

If we have more than one target, let's have more than one output.  
Action and cli shouldn't both be built at `dist`. 
I suggest naming them `review-bot` and `review-bot-cli`, respectively.

---

Could you elaborate on this? I'm not following.

What about changing it to this?
```suggestion
    "start": "node dist",
    "build": "ncc build --license LICENSE",
    "cli": "ncc build src/cli.ts && node dist",
```

My plan for this is to be a very minor QoL feature, don't want to release it.

---

> What about changing it to this?

same problem, both `cli` and action targets are built to `dist`. 
If you first do `yarn cli`, and then `yarn start`, you won't start the action. 

I suggest something like this: 

```json  
    "build": "ncc build --license LICENSE -o dist/review-bot",
    "start": "node dist/review-bot",
    "build:cli": "ncc build src/cli.ts -o dist/review-bot-cli",
    "start:cli": "node dist/review-bot-cli",
```

---

Ah makes sense!

I would like to keep the `dist` directory specifically for the review-bot.

Would `"cli": "ncc build src/cli.ts -o cli && node cli",` satisfy the requirement for you?

I would also add `/cli` to the `.gitignore`

---

Sure!
`dist-cli` is also an option =)